Gopkg.toml

```diff
@@ -12,7 +12,7 @@
 
 [[constraint]]
   name = "github.com/bitrise-tools/go-xcode"
-  branch = "master"
+  branch = "xcode-managed-codesign-group"
```

Comment:
I think this will cause some mis-understanding.

Reply:
👍